src/themes/masschallenge/collections/table.overrides

```diff
@@ -8,4 +8,9 @@
 
 .ui.selectable.table tbody tr:hover .info {
   visibility: visible;
+}
+
+.ui.noborder.table tr td {
+  border: 0px;
+  color: #8D8F94;
```

Comment:
@kitaka it is misleading to have a class called `.noborder` and then tuck in a color override as well. I recommend to remove the color override.